pkg/cli/admin/release/info.go

```diff
@@ -1165,10 +1165,98 @@ func describeReleaseDiff(out io.Writer, diff *ReleaseDiff, showCommit bool, outp
 			}
 		})
 	}
+
+	printFeatureSetSection(diff, "Default", w)
+	printFeatureSetSection(diff, string(configv1.TechPreviewNoUpgrade), w)
```

Comment:
nit: can we iterate through [`FeatureSets`][1] instead of hard-coding some well-known sets here?  If not, can the commit message explain why and how these two were selected?

[1]: https://github.com/openshift/api/blob/355cd2586ec48edab3017d1c56ed783608296f35/config/v1/types_feature.go#L147-L159

Reply:
> If not, can the commit message explain why and how these two were selected?

Yes, I'll explain.  People adding more shoudl suffer. Mayb ea dev preview.